### PR TITLE
update otel libs from v0.13.0 -> v0.20.0

### PR DESCRIPTION
**Pull request by aaron-prindle**
What is the problem being solved?
Part of #5756, adding opentelemetry trace information to skaffold commands. Updating out libs to the latest otel version adds additional useful functionality for tracing.

Why is this the best approach?
This approach uses go mod (updated via "go get <pkg>") and minor API changes to our otel API usage for the update.

What other approaches did you consider?
N/A

What side effects will this approach have?
There shouldn't be any side effects w/ this approach, the changes to otel's API were renaming/moving things (packages, functions, etc.).  The only option removed was stdout Quantile aggregation (stdout.WithQuantiles) but I do not think this will have side effects.  See https://github.com/open-telemetry/opentelemetry-go/commit/49f699d65742e144cf19b5dd28f3d3a0891bf200#diff-2b283a7fb9f9b66e31a2b51a9ae9cad3599650a633f02fea9a956c4f6a714c6c

What future work remains to be done?
N/A

Fixes: #nnn 
**Related**: _Relevant tracking issues, for context_
**Merge before/after**: _Dependent or prerequisite PRs_

**Description**

**User facing changes (remove if N/A)**

**Follow-up Work (remove if N/A)**

### CODE REVIEW

File: cmd/skaffold/skaffold.go [hunk 1]
```diff
@@ -22,16 +22,40 @@ import (
...
 
 	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel/trace"
 
 	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/color"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/instrumentation"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/stdout"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```
Comment (tejal29) [inline cmd/skaffold/skaffold.go:32]: Looks like these 3rd party libs need to be in the 3rd party import block

Comment (aaron-prindle) [reply cmd/skaffold/skaffold.go]: Thanks! This is fixed now using `gci` tool

**Comment by briandealwis** [comment]
Can you verify with `SKAFFOLD_EXPORT_TO_STDOUT` whether the quantiles is equivalent?  Or that it's ingested appropriately with Cloud Monitoring?

**Comment by aaron-prindle** [comment]
The Quantile piece is only used for the stdout exporter which is not used in data ingestion for Cloud Monitoring.  The way quantiles were used w/ the stdout exporter was not relevant to our logging setup.  This will have no import on data export/ingestion, metrics will still work as expected.  I have verified that this setup still works as expected pointing this at my own project and firing metrics there:

Screenshot 2021-04-30 3 35 39 PM

image

**Review by briandealwis — APPROVED** [review]

**Pull request merged**